admin/components.js

```diff
@@ -172,5 +186,9 @@ module.exports = {
             .command('update')
             .description('Send update device request to dashboard')
             .action(updateMetadata);
+        program
+            .command('pull-actuations')
+            .description('Fetches actuations from last 24h and executes them one by one')
```

Comment:
From last actuation pull or 24h - refine this message somehow to inform what it actually does.